sockeye/inference.py

```diff
@@ -144,15 +145,15 @@ def initialize(self, max_input_length: int, get_max_output_length_function: Call
         self.decoder_module.init_params(arg_params=self.params, aux_params=self.aux_params, allow_missing=False)
 
         if self.cache_output_layer_w_b:
-            if self.output_layer.weight_normalization:
+            if layer.weight_normalization:
```

Comment:
this seems off. please revert to original code.

Reply:
fixed